requirements.txt

```diff
@@ -1,2 +1,2 @@
 pyrogram
-asyncio-dev
\ No newline at end of file
+asyncio
```

Comment:
We really should use something like Pipenv

Reply:
I personally run it in a virtualenv is that similar lol